.github/workflows/codeql.yml

```diff
@@ -17,7 +19,8 @@ jobs:
       actions: read
       contents: read
       security-events: write
-
+    if: |
+      github.event.pull_request.user.login != 'renovate[bot]'
```

Comment:
Could we keep codeQL? there is no reason to why it should fail, but it doesn't hurt to run either IMO